package.json

```diff
@@ -137,6 +137,7 @@
     "@metamask/smart-transactions-controller": "^3.0.0",
     "@metamask/snaps-controllers": "^0.25.0",
     "@metamask/snaps-utils": "^0.25.0",
+    "@metamask/snaps-ui": "^0.26.1",
```

Comment:
Let's wait for the `0.26.1` to merge in and rebase on top of that. Otherwise we might have internal mismatches of versions.

Reply:
Shouldn't matter, but we can 🤷‍♂️